sample/src/main/java/com/mapzen/android/sample/LabelPickListenerActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.mapzen.android.sample;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.Toast;
+
+import com.mapzen.android.graphics.FeaturePickListener;
+import com.mapzen.android.graphics.LabelPickListener;
+import com.mapzen.android.graphics.MapFragment;
+import com.mapzen.android.graphics.MapzenMap;
+import com.mapzen.android.graphics.OnMapReadyCallback;
+import com.mapzen.android.graphics.model.Marker;
+import com.mapzen.tangram.LabelPickResult;
+import com.mapzen.tangram.LngLat;
+
+import java.util.Map;
+
+/**
+ * Demonstrates how to use {@link com.mapzen.android.graphics.LabelPickListener}.
+ */
+public class LabelPickListenerActivity extends AppCompatActivity {
+
+    MapzenMap map;
+
+    LabelPickListener listener = new LabelPickListener() {
+        @Override
+        public void onLabelPicked(LabelPickResult result, float positionX, float positionY) {
+            if (result == null) {
+                return;
+            }
+            double longitude = result.getCoordinates().longitude;
+            double latitude = result.getCoordinates().latitude;
+            map.addMarker(new Marker(longitude, latitude));
```

Comment:
Also it would be nice to toast the name of the label selected if this is available in the properties?